src/cryptography/hazmat/backends/interfaces.py

```diff
@@ -221,6 +221,12 @@ def elliptic_curve_exchange_algorithm_supported(self, algorithm, curve):
         Returns whether the exchange algorithm is supported by this backend.
         """
 
+    @abc.abstractmethod
+    def derive_elliptic_curve_public_point(self, private_value, curve):
```

Comment:
An excellent question. Returning the points as a tuple gives us some flexibility on how we use this in the future I suppose but does that really matter? We can always add more to the interface (or even alter it).